xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/test/java/org/xwiki/help/HelpTest.java

```diff
@@ -51,11 +55,11 @@
      * @see <a href="https://jira.xwiki.org/browse/XWIKI-13650">XWIKI-13650</a>
      */
     @Test
-    public void verifySubHeadingVelocityVariableCorrectlyEvaluatedWhenUsedInSection() throws Exception
+    void verifySubHeadingVelocityVariableCorrectlyEvaluatedWhenUsedInSection() throws Exception
     {
         // URL that we're simulating:
         //   http://localhost:8080/xwiki/bin/view/XWiki/XWikiSyntax?syntax=2.1&section=Links&xpage=print
-        setOutputSyntax(Syntax.XHTML_1_0);
+        setOutputSyntax(Syntax.HTML_5_0);
```

Comment:
We talked about making this the default. You changed your mind?

Reply:
I did some tests in this direction but redefining the output syntax a second time in the tests makes them fail.

Reply:
Ok, I can take care of that later.